.github/workflows/release.yml

```diff
@@ -23,6 +23,8 @@ jobs:
           git push --force https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git HEAD:refs/heads/v3
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - run: npx semantic-release
+      # Workaround for [semantic-release]: node version >=18 is required. Found v16.19.0.
+      # See https://github.com/semantic-release/semantic-release/blob/master/docs/support/node-version.md for more details and solutions.
+      - run: npx -p node@v18-lts -c "npx semantic-release"
```

Comment:
I would just update line 14 to this instead

```yml
          node-version: "lts/*"
```

Reply:
Good point. I didn't know about this option. I'll update PR.

Reply:
Done